saga/register-payment-connector.json

```diff
@@ -0,0 +1,19 @@
+{
+    "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
+    "tasks.max": "1",
+    "database.hostname": "payment-db",
+    "database.port": "5432",
+    "database.user": "paymentuser",
+    "database.password": "paymentpw",
+    "database.dbname" : "paymentdb",
+    "database.server.name": "dbserver2",
+    "schema.include.list": "payment",
+    "table.include.list" : "payment.outboxevent",
+    "tombstones.on.delete" : "false",
+    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "value.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "value.converter.schemas.enable": "false",
```

Comment:
same here, superfluous setting ? because no schema for plain string

Reply:
Removed.